src/gui/folderwatcher.cpp

```diff
@@ -35,6 +35,11 @@
 #include "folder.h"
 #include "filesystem.h"
 
+namespace
+{
+const char *officeLockPrefixes[] = {".~lock.", "~$"};
```

Comment:
`lockFilePrefixes`
should you not try to add more prefixes here ?
`vim` is using both a suffix and a prefix ?
should you not add support for that also ?
for example `bootstrap.sh` will have a lock file named `.bootstrap.sh.swp`

Reply:
@mgallien The task is to cover only office files, I am not sure about vim and other stuff. This should be done as a separate, bigger task

Reply:
sure but they are not prefixes (especially if what we do is checking for the existence of the substring in the whole path)